reference/3.0/Microsoft.PowerShell.Utility/ConvertFrom-Json.md

```diff
@@ -21,127 +21,128 @@ ConvertFrom-Json [-InputObject] <String> [<CommonParameters>]
-Then it uses the pipeline operator to send the delimited string to the **ConvertFrom-Json** cmdlet, which converts it to a custom object.
+The command uses Get-Content cmdlet to get the strings in a JSON file. It uses the Join operator to
+join the strings in the file into a single string that is delimited by newline characters (\`n).
+Then it uses the pipeline operator to send the delimited string to the `ConvertFrom-Json` cmdlet,
+which converts it to a custom object.
 
-The Join operator is required, because the **ConvertFrom-Json** cmdlet expects a single string.
+The Join operator is required, because the `ConvertFrom-Json` cmdlet expects a single string.
 
 ## PARAMETERS
 
 ### -InputObject
 
-Specifies the JSON strings to convert to JSON objects.
-Enter a variable that contains the string, or type a command or expression that gets the string.
-You can also pipe a string to **ConvertFrom-Json**.
+Specifies the JSON strings to convert to JSON objects. Enter a variable that contains the string,
+or type a command or expression that gets the string. You can also pipe a string to
+`ConvertFrom-Json`.
 
-The **InputObject** parameter is required, but its value can be an empty string.
-When the input object is an empty string, **ConvertFrom-Json** does not generate any output.
-The InputObject value cannot be null ($null).
+The **InputObject** parameter is required, but its value can be an empty string. When the input
+object is an empty string, `ConvertFrom-Json` does not generate any output. The **InputObject**
+value cannot be `$null`.
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases:
 
 Required: True
-Position: 1
+Position: 0
 Default value: None
 Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
 ### CommonParameters
 
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see
+[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ### System.String
 
-You can pipe a JSON string to **ConvertFrom-Json**.
+You can pipe a JSON string to `ConvertFrom-Json`.
 
 ## OUTPUTS
 
 ### PSCustomObject
 
 ## NOTES
 
-- The **ConvertFrom-Json** cmdlet is implemented by using the [JavaScriptSerializer class](https://msdn.microsoft.com/library/system.web.script.serialization.javascriptserializer).
+The `ConvertFrom-Json` cmdlet is implemented using the
+[JavaScriptSerializer class](/dotnet/api/system.web.script.serialization.javascriptserializer?view=netframework-4.7.2).
```

Comment:
4.7.2 is the latest today.  Shouldn't the ?view=* be removed so that it points to the latest version or are you trying to point to 4.7.2 specifically?

Reply:
I am going to leave this for now. I am not sure about this for .NET Core but I couldn't find the class.

Reply:
Fixed this. PSv6+ uses Newtonsoft JSON.net. Updated links.